guides/doc-Upgrading_Disconnected_Project/master.adoc

```diff
@@ -17,11 +17,11 @@ include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[le
 // Upgrading Project Server overview
 include::common/modules/con_upgrading-overview.adoc[leveloffset=+1]
 
-include::common/modules/con_upgrading-prerequisites.adoc[leveloffset=+2]
+include::common/modules/con_upgrading-prerequisites.adoc[leveloffset=+1]
```

Comment:
We can follow Lena's suggestion even here and in other places. https://github.com/theforeman/foreman-documentation/pull/2599#discussion_r1440384082

Reply:
I meant it only for the snippet, I don't think it's necessary otherwise.